src/components/RoomsCheckbox.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="filter__rooms">
+    <div class="filter__rooms__title title">комнаты</div>
+    <div class="filter__rooms__buttons">
+      <label v-for="room in roomsValues" :key="room">
+        <input type="checkbox" :value="room" v-model="selectedRooms"
+        >
+        <span
+            class="filter__rooms__checkbox"
+            :class="{ selected: isSelected(room) }">
+          {{roomsValuesLayout[room]}}
+        </span>
+      </label>
+    </div>
+  </div>
+</template>
+
+<script setup>
+
+  const selectedRooms = defineModel('selectedRooms');
+
+  const roomsValues = ['s', '1', '2', '3'];
```

Comment:
ну почему дублируешь код ? 
roomsValuesLayout
```suggestion
  const roomsValues = ObjectKeys(roomsValuesLayout);
```